admin/acceptance.py

```diff
@@ -70,11 +72,13 @@ def run_tests(reactor, nodes, control_node, agent_nodes, trial_args):
         agent, to run API acceptance tests against.
     :param list trial_args: Arguments to pass to trial. If not
         provided, defaults to ``['flocker.acceptance']``.
+    :param FilePath certificates_path: Directory where certificates can be
+        found.
```

Comment:
I think this will be a directory where `flocker-ca --create-control-certificate` has been run?  Maybe mention that here.